content/hardware/05.pro-solutions/solutions-and-kits/portenta-machine-control/tutorials/plc-ide-setup-license/content.md

```diff
@@ -100,21 +100,54 @@ Click OK to save the settings and press the connect button on **On-line > connec
 
 ![Connect to the board](assets/connect.png)
 
-### 6. License Activation
+### 6. License Activation With Product Key (Portenta Machine Control)
```

Comment:
```suggestion
### 6. License Activation With Product Key (Portenta Machine Control)
```
Long but ok!, we will need to find a new way to say this once we have more compatible products :)

Reply:
It's ok!